test/extended/prometheus/prometheus.go

```diff
@@ -929,13 +927,18 @@ func findMetricLabels(f *dto.MetricFamily, labels map[string]string, match strin
 	return result
 }
 
-func getBearerTokenURLViaPod(ns, execPodName, url, bearer string) (string, error) {
-	cmd := fmt.Sprintf("curl -s -k -H 'Authorization: Bearer %s' %q", bearer, url)
-	output, err := e2eoutput.RunHostCmd(ns, execPodName, cmd)
+func getBearerTokenURLViaPod(oc *exutil.CLI, execPodName, url, bearer string) (string, error) {
```

Comment:
Don't declare this function in two different files, please refactor and declare it once as a public function (GetBearerTokenURLViaPod) in perhaps test/extended/util/prometheus/helpers.go, then import the test/extended/util/prometheus package to use it where you need it.

Reply:
Moved the function to helpers and imported it in both places